apps/admin-ui/components/Layout/Header/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Divider } from "@mui/material";
+import Breadcrumb from "./Breadcrumbs";
+import { useRouter } from "next/router";
+import LanguageSwapper from "../LanguageSwapper";
+
+export interface BreadcrumbsNameMaps {
+    title: string;
+    links: { [key: string]: string }
+}
+export default function Header() {
+    const { pathname } = useRouter()
+
+    const breadcrumbsNameMap: BreadcrumbsNameMaps[] = [
+        {
+            title: 'Mass Offer',
+            links: {
+                '/massOffer': 'Mass offer',
+            }
+        },
+        {
+            title: 'Masses',
+            links: {
+                '/masses': 'Masses',
+                '/masses/create': 'Create',
+                '/masses/modify': 'Modify',
+                '/masses/delete': 'Cancel',
+            }
+        },
+        {
+            title: 'Finances',
+            links: {
+                '/finances': 'Finances',
+                'finances/widrawal': 'Widrawal',
+            }
+        },
+    ]
```

Comment:
if the application intends to translate text, then even the breadcrumbs should be translated. 
this means, the `title` and `link` values should be translated